encoder/vaapiencoder_base.cpp

```diff
@@ -190,8 +190,8 @@ YamiStatus VaapiEncoderBase::setParameters(VideoParamConfigType type, Yami_PTR v
         VideoParamsCommon* common = (VideoParamsCommon*)videoEncParams;
         if (common->size == sizeof(VideoParamsCommon)) {
             PARAMETER_ASSIGN(m_videoParamCommon, *common);
-            if(m_videoParamCommon.rcParams.bitRate > 0)
-	         m_videoParamCommon.rcMode = RATE_CONTROL_CBR;
+            //if(m_videoParamCommon.rcParams.bitRate > 0)
```

Comment:
@lizhong1008, I think we should not to bring confusion here. If user set bitrate and qp but forget the mode, which mode we should choose?